build_guides/bluetooth_conversion.md

```diff
@@ -0,0 +1,103 @@
+---
+title: From Wired to Bluetooth
+description: How To Add Bluetooth to Most Keyboards
+author: Boardsource
+subcategory: easy
+draft: false
```

Comment:
this should be true I like its seems strange but just a incase something gets merged before its ready for the website this will stop it from being seen. More of a best practice.